runtime/compiler/z/codegen/S390J9CallSnippet.cpp

```diff
@@ -213,14 +214,14 @@ TR::S390J9CallSnippet::generateInvokeExactJ2IThunk(TR::Node * callNode, int32_t
       cursor += 2;
 
       TR::SymbolReference *helper = cg->symRefTab()->findOrCreateRuntimeHelper(TR_methodHandleJ2IGlue, false, false, false);
-      uintptrj_t destAddr = (uintptrj_t)helper->getMethodAddress();
+      intptrj_t destAddr = (uintptrj_t)helper->getMethodAddress();
```

Comment:
nitpick, the cast is `(uintptrj_t)` but the type of `destAddr` is `intptrj_t`.

Reply:
Not a nit pick.  Thanks for pointing that out.  I'll fix and re-push.